docs/source/examples/nn/representation/combination.py

```diff
@@ -0,0 +1,45 @@
+"""Demonstrate creating a model with a combine representation."""
+
+import torch
+
+from pykeen.models import ERModel
+from pykeen.nn import CombinedRepresentation, ConcatProjectionCombination, Embedding
+from pykeen.triples.generation import generate_triples_factory
+from pykeen.typing import FloatTensor
+
+random_state = torch.manual_seed(0)
```

Comment:
Those unfortunately do not generalize well to multiple device (i.e., the random state on CUDA is of a different class).

Reply:
interesting! I guess the other examples didn't include something for random control, so i'll just remove this entirely.

Reply:
cf4f1772